### ISSUE TITLE
Any way to fetch a specific page?

### ISSUE THREAD

**Issue by MarcelloTheArcane** [Airtable/airtable.js#168]
I am trying to create a forwarding proxy from airtable to a [custom integration field in prismic](https://intercom.help/prismicio/en/articles/1401183-connect-to-a-custom-api).

They have a limit of maximum 50 records at a time, so I want to forward the pagination directly to Airtable. 

Is there a way I can select which page of data is fetched? Currently the only way I can see would be to fetch the entire table from Airtable and drop the pages I don't want, which doesn't sound very optimal.

I'm wondering if something like this is possible:

```js
base('Table')
    .select({
      maxRecords: 5,
      sort: [{
        field: 'Last Modified',
        direction: 'desc'
      }],
      page: 3 // ??
    })
```

**Comment by ryanpwaldon** [comment]
Did you find a way to achieve this by any chance @MarcelloTheArcane?

**Comment by MarcelloTheArcane** [comment]
No I didn't, @ryanpwaldon 🙁 

I ended up just using `.slice()` to fit the data into the right page.

**Comment by ryanpwaldon** [comment]
Yeah it's a bit annoying haha. I guess cursor based pagination is here to stay. I ended up using Gridsome which solves this by importing all the data from Airtable and storing it in its own queryable data store. Gridsome is only suitable for SSG, but you could probably replicate their method for other use cases.
